src/components/Dashboard/Content/index.tsx

```diff
@@ -26,6 +28,18 @@ const DashboardContent = ({
   onNextSurvey,
   onIndicatorTapped,
 }: DashboardContentProps): JSX.Element => {
+  const getCurrentSurvey = (): Survey | undefined => {
+    return _get(surveys, currentPosition, undefined);
```

Comment:
we don't need the `undefined` as it's the default one.
```suggestion
    return _get(surveys, currentPosition);
```